modules/ocf_hpc/manifests/compute.pp

```diff
@@ -2,11 +2,12 @@
   require ocf_hpc
 
   # install proprietary nvidia drivers and CUDA.
-  ocf::repackage {
-    ['nvidia-driver', 'nvidia-settings']:
+  ocf::repackage { ['nvidia-driver', 'nvidia-settings', 'nvidia-cuda-toolkit']:
```

Comment:
So we decided to incompatibly bump the CUDA version after all?

Reply:
I tried to downgrade Nvidia drivers on corruption and ended up being forced to upgrade the cuda version :/

Reply:
Why were you downgrading drivers?

Reply:
Tensorflow wasn't working, so I thought it might have been a bug introduced when we upgraded drivers (though it turned out to be that a kernel module wasn't loaded).

Reply:
Why can't we downgrade the CUDA version again?

Reply:
I (very unwisely) tried removing the nvidia drivers to downgrade them from the stretch-backports to the stretch version, and ended up with a bunch of broken packages; when I put everything back, I couldn't install CUDA from stretch because of package conflicts with the stretch-backports nvidia drivers. So I'm not sure, to be honest.

Reply:
does it really matter if we're using CUDA from stretch-backports instead of stretch? If the newer drivers work we should just stick with them.

Reply:
Our old versions of Tensorflow are compatible with the new CUDA version (9.1), and we can upgrade to the newest versions of TF now, so I don't think we'll have any problems with using the stretch-backports version.

Reply:
just upgrade everything to the latest version that works and shipit

Reply:
But are any HPC users relying on, or having binaries compiled against, CUDA 8? If so we'll break their code.

Reply:
Since corruption is already on CUDA 9, and nobody has complained yet, probably not.